packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -98,9 +99,9 @@ class _MyHomePageState extends State<MyHomePage> {
           // arranges them vertically. By default, it sizes itself to fit its
           // children horizontally, and tries to be as tall as its parent.
           //
-          // Invoke "debug paint" (press "p" in the console where you ran
-          // "flutter run", or select "Toggle Debug Paint" from the Flutter tool
-          // window in IntelliJ) to see the wireframe for each widget.
+          // Invoke "debug painting" (press "p" in the console or choose the
+          // "Toggle Debug Paint" command in Android Studio or Visual Studio Code)
```

Comment:
We have discovery issues with this action, so its likely worth mentioning that it's found in the Inspector window.